src/Console/Database/CheckSchemaIntegrityCommand.php

```diff
@@ -118,11 +125,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $checker = new DatabaseSchemaIntegrityChecker(
             $this->db,
             $input->getOption('strict'),
-            $input->getOption('ignore-innodb-migration'),
-            $input->getOption('ignore-timestamps-migration'),
-            $input->getOption('ignore-utf8mb4-migration'),
-            $input->getOption('ignore-dynamic-row-format-migration'),
-            $input->getOption('ignore-unsigned-keys-migration')
+            !$input->getOption('check-innodb-migration'),
```

Comment:
```suggestion
            !$input->getOption('check-innodb-migration') && !$input->getOption('check-all-migrations'),
```

Needed to handle checking all migrations?

Reply:
Indeed.